packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import Button from 'terra-button';
+import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
+
+const Icon = <IconCalendar />;
+
+const propTypes = {
```

Comment:
None of these are really required. If no props are provided, it would just render a date picker with no default date.